pkg/restapi/operation/operations.go

```diff
@@ -310,6 +340,25 @@ func (vc *VaultCollection) createDataVault(vaultID string) error {
 	return nil
 }
 
+// storeDataVaultConfiguration stores a given DataVaultConfiguration
```

Comment:
Generally when it comes to unexported methods, the best way to document them is to give them good names that makes it obvious what they're doing. When that's not possible/very difficult, then you resort to comments.

In this case, I would argue that the name makes it pretty clear what it's doing. `storeDataVaultConfiguration stores a given DataVaultConfiguration` is redundant here.

(Note that sometimes it's hard to escape adding these "obvious comments" since the linter forces us to add comments to all exported methods and functions... but in this case, you have some more options).

Just something to consider...